app/src/main/java/com/sapuseven/untis/data/databases/UserDatabase.kt

```diff
@@ -406,7 +427,8 @@ class UserDatabase private constructor(context: Context) : SQLiteOpenHelper(cont
 		val masterDataTimestamp: Long,
 		val userData: UntisUserData,
 		val settings: UntisSettings? = null,
-		val created: Long? = null
+		val created: Long? = null,
+		val bookmarks: ArrayList<TimetableBookmark>
```

Comment:
Use generic list type for declaration
```suggestion
		val bookmarks: List<TimetableBookmark>
```